pkg/volume/util/volumehelper/volumehelper.go

```diff
@@ -151,3 +151,13 @@ func GetVolumeMode(volumeSpec *volume.Spec) v1.PersistentVolumeMode {
 	// override the value as filesystem for the backward compatibility.
 	return v1.PersistentVolumeFilesystem
 }
+
+// GetPersistentVolumeClaimVolumeMode returns VolumeMode. If no VolumeMode was
+// requested, it returns "Filesystem" VolumeMode by default.
+func GetPersistentVolumeClaimVolumeMode(claim *v1.PersistentVolumeClaim) v1.PersistentVolumeMode {
```

Comment:
We should not default any values now that the API is defaulted.

Reply:
the defaulting is only applied if the feature gate is on, right? so this field could be nil

Reply:
Yes there needs to be a nil check.  But if it's nil, we should not default a value.